src/cdk/a11y/key-manager/list-key-manager.ts

```diff
@@ -274,6 +274,11 @@ export class ListKeyManager<T extends ListKeyManagerOption> {
     return this._activeItem;
   }
 
+  /** Whether the user is currently typing into the manager using the typeahead feature. */
+  get isTyping(): boolean {
```

Comment:
Could this be a method?

Reply:
It could. I made it a getter because we have a couple of other similar properties on the `ListKeyManager` that are getters.

Reply:
I would prefer to make this one a method since it's very obviously a computed value